go.mod

```diff
@@ -1,9 +1,10 @@
 module github.com/mackerelio-labs/terraform-provider-mackerel
 
-go 1.21
+go 1.22
```

Comment:
In Go 1.21 or higher versions, this means... "Go 1.22 development version", so `1.22 < 1.22rc1, 1.22.0, 1.22.1...`

So, in my opinion for now, it's better to describe `go 1.22.0` and `go-version: 1.22.x` (not go-version-file.)

Reply:
Thank you, I fixed it: https://github.com/mackerelio-labs/terraform-provider-mackerel/pull/212/commits/a4c42f7b6b6d6ae1fd4bd9614b5c6c9c61e35392